### ISSUE TITLE
Random Seed

### ISSUE THREAD

**Issue by GMarzinotto** [luheng/deep_srl#13]
Hello, 

First thank you very much for this contribution!

However, when I try to train the model from scratch  
I'm getting, lower performance. (about 1pt of f-score)
I'm able to download your model and reproduce de results in your paper but I'm not able to retrain it.

I believe this may be due to a difference in the random seed, a different hyper-parameter configuration or a difference in software/hardware.

The configurations and random seed provided in this repo are the same used in the paper?  

Kind regards,
Gabriel M
